app/Models/Device.php

```diff
@@ -464,6 +464,17 @@ public function scopeIsNotDisabled($query)
         ]);
     }
 
+    public function scopeWhereAttributeDisabled($query, $attribute)
```

Comment:
We should start type-hinting on new functions
```suggestion
    public function scopeWhereAttributeDisabled(Builder $query, string $attribute) : Builder
```

Reply:
No idea what the convention is.